### ISSUE TITLE
Sample issue for training on August 14

### ISSUE THREAD

**Issue by GeorgeKerscher** [GeorgeKerscher/Hello-World#10]
To have an issue that people attending the training can play with, I created this one. 

I suggest that folks add a comment. When viewing the issue, screen reader users can hit "e" for edit and you should be taken to the edit field to enter a comment. Go into edit mode in NVDA and in Jaws get out of virtual cursor. You can then enter a comment. Go into browse mode in NVDA or in Jaws activate the virtual cursor and hit "b" for button, and select the add comment.

Good luck!!!

**Comment by GeorgeKerscher** [comment]
I am following my own suggestions and am adding a comment. When you use the button, the first one is to close the comment, but you want to select the second one and add a comment.

**Comment by hillikera** [comment]
Testing my first comment. Hello hello hellooooooo world!

**Comment by GeorgeKerscher** [comment]
Hi Allison,

You should get this message in your inbox. I am responding on GitHub.  If you reply to the email message, it goes into a comment. If you got directly to GetHub, just hit "e" to go to the edit field and leave another comment.

**Comment by ljmaher** [comment]
Louis Maher test comment.

**Comment by CodyCare** [comment]
This is my comment. There are many like it, but this one is mine.

**Comment by GeorgeKerscher** [comment]
Hi Cody,

I got an email with your comment. I clicked on view it on GitHub. I then just hit e and am taken to the edit field.

Yes, the first comment is the description. If you create a new issue, you put in the title, and then go to the comment and the first comment serves as the description.

Give it a try. I would get a notification in email of the new issue.
